tectonic/msvc-include/unistd.h

```diff
@@ -0,0 +1,2 @@
+// For now, this is a total stub.
+// See https://stackoverflow.com/q/341817 for some things we could want.
```

Comment:
I’d guess it’d be more common to use a HAVE_UNISTD_H flag which is undefined on Windows, but I’m not really sure. I’m not a C/C++ developer and cannot speak for their conventions!